internal/utils.go

```diff
@@ -95,3 +97,23 @@ func CRC32hash(input []byte) int {
 	return int(crc32.Checksum(input, crc32Table))
 
 }
+
```

Comment:
```suggestion
// We run the first tick immediately, then set the interval
```

Reply:
(This was a suggestion to leverage godoc for this, but I must admit I really suck with godoc itself)

Reply:
fixed